cddl/ccf-receipt.cddl

```diff
@@ -0,0 +1,50 @@
+ccf-cose-root-signature-tagged = #6.18(ccf-cose-root-signature)
```

Comment:
Just to make sure, have you searched for a cddl include/import or similar?

I don't find any in the standard though...

Reply:
Yes I did, it is being designed as far as I can tell, but it is not ready to use yet: https://datatracker.ietf.org/doc/draft-ietf-cbor-cddl-modules/